cmd/prysmctl/testnet/generate_genesis.go

```diff
@@ -265,6 +285,39 @@ func generateGenesis(ctx context.Context) (*ethpb.BeaconState, error) {
 	if err != nil {
 		return nil, err
 	}
+	if overrideEth1Data {
+		log.Print("Overriding Eth1Data with data from execution client")
+		conn, err := rpc.Dial(generateGenesisStateFlags.ExecutionEndpoint)
+		if err != nil {
+			return nil, errors.Wrapf(
+				err,
+				"could not dial %s please make sure you are running your execution client",
+				generateGenesisStateFlags.ExecutionEndpoint)
+		}
+		client := ethclient.NewClient(conn)
+		header, err := client.HeaderByNumber(ctx, nil)
```

Comment:
Lets do 0 here to be safe, using `nil` returns the current head. 
```suggestion
		header, err := client.HeaderByNumber(ctx, nil)
```